assets/styles.scss

```diff
@@ -0,0 +1,96 @@
+---
+title: false
+styles: true
+---
+
+@import "alembic";
+
+// Notification bar
+.notification-bar {
+  text-align: center;
+  padding: .75rem 1.25rem;
+  margin-bottom: 1rem;
+  background-color: $codeBackgroundColour;
```

Comment:
Hmm, so I tried changing the color here to `"red"` and `#f00` without success. Is it possible to make the banner background orange (or dark-grey) to stand out? What am I missing?

Reply:
@odow strange you couldn't change it, guessing you were using the correct build commands? I'll change it to a tint of orange.